### DISCUSSION TITLE
GPU memory usage

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7667] [Q&A]
<img width="335" alt="image" src="https://user-images.githubusercontent.com/87828192/128086559-b7bd2d1f-6f18-4bf2-b195-f5f6ee909cd5.png">

My PIXI app is using 400mb GPU memory.
Is it ok? or Is it too high?

**Comment by ivanpopelyshev** [comment]
the usual :)
